v2/devices/yggdrasil.yml

```diff
@@ -194,111 +194,91 @@ operating_systems:
           - core:user_action:
               action: "recovery"
     slideshow: []
-  - name: "VollaOS"
-    options:
-      - var: "install_twrp"
-        name: "TWRP"
-        tooltip: "Install Team Win Recovery Project (TWRP) recovery"
-        type: "checkbox"
-        value: true
-    prerequisites:
-      - "untested"
+  - name: "Volla OS"
+    prerequisites: []
     steps:
       - actions:
           - core:download:
-              group: "VollaOS"
+              group: "firmware"
               files:
-                - url: "https://ota.volla.tech/volla-9.0-20201019-nightly-k63v2_64_bsp-signed.zip"
+                - url: "https://volla.tech/filedump/vollaos-10.0-ubports.zip"
```

Comment:
Perhaps you should add some versioning to filename in URL. This way we should be able to update it later without breaking checksum of existing config.

Reply:
done